docs/_ext/single_sourced_data.py

```diff
@@ -76,6 +76,11 @@ def _nodes_from_rst(
     """Turn an RST string into a list of nodes.
 
     These nodes can be used in the document.
+
+    :param state: The superclass for a docutils statemachine which contains a list or transitions
+        and transition methods
+    :param rst_source: The source of the RST content from which nodes will be extracted
```

Comment:
While it's called RST, really it'd work with other integrated sources, like MyST Markdown. But I didn't know that at the time of the initial implementation.